packages/queued-request-controller/package.json

```diff
@@ -33,9 +33,9 @@
     "@metamask/base-controller": "^3.2.3",
     "@metamask/controller-utils": "^5.0.2",
     "@metamask/json-rpc-engine": "^7.3.0",
-    "@metamask/network-controller": "^15.2.0",
+    "@metamask/network-controller": "^16.0.0",
     "@metamask/rpc-errors": "^6.1.0",
-    "@metamask/selected-network-controller": "^3.1.1",
+    "@metamask/selected-network-controller": "^3.1.2",
```

Comment:
Should we add this to the changelog too?

Reply:
Fixed: https://github.com/MetaMask/core/commit/194c74e747285515e41738387b9b59d156b9b0fe